src/ape/managers/converters.py

```diff
@@ -119,14 +180,18 @@ def __repr__(self):
 
     @cached_property
     def _converters(self) -> Dict[Type, List[ConverterAPI]]:
+        list_tuple_converter.converter = self  # Splice this in (no need for the rest)
         converters: Dict[Type, List[ConverterAPI]] = {
             AddressType: [address_api_converter, hex_address_converter],
             bytes: [hex_converter],
             int: [],
+            Decimal: [],
+            list: [list_tuple_converter],
```

Comment:
Should these also be `List` and `Tuple` (from `typing` module)?

Reply:
these are concrete types, e.g. `convert(v, list)`

Tried to think of what `convert(v, List[int, str, ..., bytes])` might look like, but it seemed bad UX

Reply:
Yeah that makes sense.